lib/import/colorectal/providers/st_george/st_george_handler_colorectal.rb

```diff
@@ -0,0 +1,317 @@
+require 'possibly'
+require 'pry'
+require 'date'
+
+module Import
+  module Colorectal
+    module Providers
+      module StGeorge
+        # Process St George-specific record details into generalized internal genotype format
+        class StGeorgeHandlerColorectal < Import::Germline::ProviderHandler
+          include Import::Helpers::Colorectal::Providers::Rj7::Constants
+
+          def process_fields(record)
+            genotype = Import::Colorectal::Core::Genocolorectal.new(record)
+
+            # records using new importer should only have SRIs starting with V
+            return unless record.raw_fields['servicereportidentifier'].start_with?('V')
+
+            # add standard passthrough fields to genotype object
+            genotype.add_passthrough_fields(record.mapped_fields, record.raw_fields,
+                                            PASS_THROUGH_FIELDS)
+
+            assign_test_type(genotype, record)
+            genotype = assign_test_scope(genotype, record)
```

Comment:
I don't think you need to reassign `genotype` here, it's being mutated in the `assign_test_scope` method.

Reply:
removed in  [b64fd4c](https://github.com/NHSDigital/data_management_system/commit/b64fd4c1b851661d098425a1ab1df681db92063f)

Reply:
removed reassignment in [075175e](https://github.com/NHSDigital/data_management_system/commit/075175e335e5a81d699eebf4501ad28f9d5a8048)